airflow_provider_hightouch/sensors/hightouch.py

```diff
@@ -0,0 +1,91 @@
+from typing import Optional
+
+from airflow.sensors.base import BaseSensorOperator, BaseOperatorLink
+
+from airflow.exceptions import AirflowException
+from airflow.utils.decorators import apply_defaults
+
+from airflow_provider_hightouch.hooks.hightouch import HightouchHook
+from airflow_provider_hightouch.utils import parse_sync_run_details
+
+from airflow_provider_hightouch.consts import (
+    TERMINAL_STATUSES
+)
+
+
+class HightouchLink(BaseOperatorLink):
+    name = "Hightouch"
+
+    def get_link(self, operator, dttm):
+        return "https://app.hightouch.io"
+
+
+class HightouchMonitorSyncRunOperator(BaseSensorOperator):
```

Comment:
I would rename this class to a Sensor, maybe `HightouchSyncRunSensor`  or simply `HightouchSyncSensor` to follow Airflow conventions